hsctl

```diff
@@ -274,7 +274,7 @@ migrate_hs() {
 manage_py_hs() {
     echo "MANAGEPY:"
     echo "  - docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}"
-    docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}
+    docker exec -u hydro-service hydroshare python manage.py ${@:2}
 }
```

Comment:
I'm unfamiliar with use of the "at symbol" `@` in bash can you explain or link to a description?

Reply:
that could be a good link to explain: https://unix.stackexchange.com/questions/92978/what-does-this-2-mean-in-shell-scripting